lib/compress.js

```diff
@@ -4858,6 +4858,12 @@ merge(Compressor.prototype, {
             }
             break;
         }
+        // void 0 == x => null == x
+        if (compressor.option("comparisons") &&
+            (self.operator == "==" || self.operator == "!=") &&
```

Comment:
Please put the `&&`'s out front to match the coding style above.

Reply:
Sure. (The code style isn't particularly consistent - I find both `&&` and `||` on the end of the line in the same file.)